src/main/scala/mesosphere/marathon/state/AppDefinition.scala

```diff
@@ -331,80 +306,4 @@ object AppDefinition {
 
   def fromProto(proto: Protos.ServiceDefinition): AppDefinition =
     AppDefinition().mergeFromProto(proto)
-
-  protected[marathon] class WithTaskCountsAndDeployments(
-    appTasks: Seq[EnrichedTask],
-    healthCounts: HealthCounts,
-    runningDeployments: Seq[DeploymentPlan],
-    private val app: AppDefinition)
-      extends AppDefinition(
-        app.id, app.cmd, app.args, app.user, app.env, app.instances, app.cpus,
-        app.mem, app.disk, app.executor, app.constraints, app.uris,
-        app.storeUrls, app.ports, app.requirePorts, app.backoff,
-        app.backoffFactor, app.maxLaunchDelay, app.container,
-        app.healthChecks, app.dependencies, app.upgradeStrategy,
-        app.labels, app.acceptedResourceRoles, app.version) {
-
-    /**
-      * Snapshot of the number of staged (but not running) tasks
-      * for this app
-      */
-    @JsonProperty
-    val tasksStaged: Int = appTasks.count { eTask =>
```

Comment:
I think that this property as a lazy val or something like that is generally useful. Not only in the API.

But of course, we can recreate it if we actually use it.